src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -63663,14 +63663,14 @@ public CustomHandler(int literalLength, int formattedCount, string s" + (validit
 
         [Theory]
         [CombinatorialData]
-        public void StringInterpolation_05(bool useBoolReturns, bool validityParameter)
+        public void StringInterpolation_05(bool useBoolReturns, bool validityParameter, [CombinatorialValues(@"$""{s}""", @"$""{s}"" + $""""")] string expression)
```

Comment:
Could we also test the following with nullability?
- `$"..." + null`
- longer string additions (nested binary ops)

Reply:
The former is not an interpolated string composed entirely of interpolated strings, and is not subject to a handler conversion. I can add more nested ops.